webui/react/src/styles/antd.scss

```diff
@@ -139,6 +139,7 @@ html {
   }
   .ant-spin-nested-loading {
     height: 100%;
+    width: 100%;
   }
   .ant-spin-nested-loading > div > .ant-spin-spinning {
     max-height: inherit;
```

Comment:
I propose styling the spinner text a little be differently so it looks less like a link (added below line 146):
```
  .ant-spin-nested-loading .ant-spin-text {
    color: var(--theme-colors-monochrome-8);
    font-style: italic;
    margin-top: var(--theme-sizes-layout-big);
  }
```

Reply:
this is strictly better. thanks